MainDemo.Wpf/Domain/MainWindowViewModel.cs

```diff
@@ -286,6 +286,14 @@ private static IEnumerable<DemoItem> GenerateDemoItems(ISnackbarMessageQueue sna
                     DocumentationLink.StyleLink("ListView")
                 });
 
+            yield return new DemoItem(
+                "Tabs",
+                typeof(Tabs),
+                new[]
+                {
+                    DocumentationLink.DemoPageLink<Tabs>()
```

Comment:
Might be nice to include the link to the tab control resource dictionary.
```suggestion
                    DocumentationLink.DemoPageLink<Tabs>(),
                    DocumentationLink.StyleLink("TabControl")
```